### PR TITLE
Fix CI workflows running the Dockerfile

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

This is an attempt to fix the CI workflows running our Dockerfile.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Commit-by-commit.

### CODE REVIEW

File: .github/workflows/vast.yaml [hunk 1]
```diff
@@ -66,23 +66,21 @@ jobs:
           apt-get -qqy install git libsnappy-dev libbrotli-dev zlib1g-dev wget liblz4-dev libzstd-dev ninja-build libpcap-dev libssl-dev libbenchmark-dev tcpdump lsb-release libflatbuffers-dev flatbuffers-compiler-dev libyaml-cpp-dev libsimdjson-dev gnupg2 gcc-8 g++-8 python3 python3-pip python3-venv jq apt-transport-https ca-certificates curl gnupg-agent software-properties-common
 
           # Need to specify backports explicitly, since spdlog and fmt also have
-          # regular buster packages.
-          apt-get -qqy -t buster-backports install libspdlog-dev libfmt-dev
+          # buster packages. Also, this comes with a newer version of CMake.
+          apt-get -qqy -t buster-backports install cmake libspdlog-dev libfmt-dev
 
-          # clang 9 (c.f. https://apt.llvm.org/)
+          # clang-19 (c.f. https://apt.llvm.org/)
```
Comment (tobim) [inline .github/workflows/vast.yaml:72]: Typo.
```suggestion
          # clang-10 (c.f. https://apt.llvm.org/)
```

**Comment by 0snap** [comment]
@dominiklohmann see also [line 86](https://github.com/tenzir/vast/blob/b41d525459d15289786cd22108e778479d4e21f4/Dockerfile#L86) in the Dockerfile. Bumping Arrow requires a modification there, too.

**Comment by 0snap** [comment]
I'm not a fan of removing `qq` from all `apt` invocations. It clutters the logs.

**Review by 0snap — APPROVED** [review]
Looks alright to me, CI passes. Can you explain why you want verbose logging for `apt`, @dominiklohmann ?

**Comment by dominiklohmann** [comment]
> I'm not a fan of removing `qq` from all `apt` invocations. It clutters the logs.
> Looks alright to me, CI passes. Can you explain why you want verbose logging for `apt`, @dominiklohmann ?

The qq hid the issue from the logs, which would've been visible if we didn't suppress them.

I only removed it from the CI workflow, not from the Dockerfile itself, and I think for CI we want all the verbosity we can get.

**Pull request merged**